card.js

```diff
@@ -0,0 +1,37 @@
+const projects = [
+  {
+    id: 1,
+    title: 'Tonic',
+    image: 'images/portfolio-1.svg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['html', 'css', 'javaScript'],
+    source: 'https://github.com/josuerushanika/Josue-portfolio',
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories',
+    image: 'images/portfolio-2.svg',
+    description: 'A daily selectin of privalety personalized reads; no accounts or sign-ups required',
```

Comment:
- [ ] on the desktop version, please ensure that the buttons are aligned with the image on the right and are of thesame size like in the original version
yours
![Screenshot (719)_LI](https://user-images.githubusercontent.com/65298048/210846412-b8f04ea0-ffcf-4ffa-b263-b55b0464d759.jpg)
figma
![Screenshot (720)](https://user-images.githubusercontent.com/65298048/210846516-240f2ba8-7376-418a-b921-abd5fff7d917.png)

Reply:
This is fixed